### PR TITLE
feat: support global installation

### PR DESCRIPTION
**Pull request by wilr**

### CODE REVIEW

File: bin/serve [hunk 1]
```diff
@@ -6,22 +6,35 @@ use SilverStripe\Core\Environment;
...
     __DIR__ . "/../vendor/autoload.php", // in ./bin
     __DIR__ . "/../../../autoload.php", // in ./vendor/silverstripe/serve/bin
 ];
+
+
+if (!empty($options['path'])) {
+    $options['path'] = realpath($options['path']);
+    $dir = rtrim($options['path'], '/');
+
+    if (is_dir($dir)) {
```
Comment (emteknetnz) [inline bin/serve]:
```suggestion
    if (!is_dir($dir)) {
```
?

File: bin/serve [hunk 2]
```diff
@@ -6,22 +6,35 @@ use SilverStripe\Core\Environment;
 
 define('BASE_URL', '');
 
+// Parse command-line options and env vars
+$options = getopt(null, [ 'host:', 'port:', 'hash:', 'bootstrap-file:', 'open', 'path:' ]);
```
Comment (emteknetnz) [inline bin/serve]:
```suggestion
$options = getopt('', [ 'host:', 'port:', 'hash:', 'bootstrap-file:', 'open', 'path:' ]);
```
https://www.php.net/manual/en/function.getopt.php

Can't pull null to string params in php8.1, it'll result in a deprecation warning

**Review by emteknetnz — CHANGES_REQUESTED** [review]

File: bin/serve (hunk 1 above)
Comment (wilr) [reply bin/serve]: Updated

File: bin/serve (hunk 2 above)
Comment (wilr) [reply bin/serve]: Updated

**Review by emteknetnz — APPROVED** [review]
Tested locally, can confirm it works

**Comment by emteknetnz** [comment]
[2.4.0](https://github.com/silverstripe/silverstripe-serve/releases/tag/2.4.0)

**Pull request merged**